### PR TITLE
Update VirtualBox troubleshooting paragraph

### PR DESCRIPTION
**Pull request by kyrios123**
Add some clarifications in the troubleshooting section for virtualbox.

Signed-off-by: Pierre-Yves <pyu@riseup.net>

### CODE REVIEW

**Comment by kyrios123** [comment]
@JoshStrobl If this is accepted, could you please publish this update asap ?  It's a quiet frequent error with virtualbox and I saw some people trying to do some crazy things to attempt to fix it.

Thanks!

File: software/virtualbox/en.md [hunk 1]
```diff
@@ -26,11 +26,24 @@ virtualbox | virtualbox-current
 ----- | -----
 VirtualBox modules for the **linux-lts** kernel | VirtualBox modules for the **linux-current** kernel
 
+You **MUST** reboot your computer before running VirtualBox for the first time
+
 ### Troubleshooting
-If the wrong VirtualBox package was installed, the following error will pop up when starting a virtual machine. Please check the instructions just above to install the correct package for your kernel.
+The Error `Kernel driver not installed (rc=-1908)` may occur if
```
Comment (JoshStrobl) [inline software/virtualbox/en.md]: This makes sense to go below the image where it's closer to the text.

File: software/virtualbox/en.md [hunk 2]
```diff
@@ -26,11 +26,24 @@ virtualbox | virtualbox-current
 ----- | -----
 VirtualBox modules for the **linux-lts** kernel | VirtualBox modules for the **linux-current** kernel
 
+You **MUST** reboot your computer before running VirtualBox for the first time
```
Comment (JoshStrobl) [inline software/virtualbox/en.md]: Having the bold should be enough, probably no need for it to be capitalized.

**Comment by kyrios123** [comment]
Requested changes applied!

**Pull request merged**